src/useAsync/useAsync.ts

```diff
@@ -28,22 +28,22 @@ export type AsyncState<Result> =
 
 export interface UseAsyncActions<Result, Args extends unknown[] = unknown[]> {
   /**
-   * Reset state to initial, when async function haven't been executed.
+   * Reset state to initial, when the async function hadn't been executed.
```

Comment:
hadn't -> hasn't?

Or maybe we can just remove the `when the async function hadn't been executed.` part altogether

Reply:
I went with deleting that part altogether.